orchestra/actions/util.py

```diff
@@ -38,24 +46,143 @@ def run_script(script,
 
     script_to_run += script
 
-    if quiet:
+    if capture_output:
         stdout = subprocess.PIPE
         stderr = subprocess.PIPE
     else:
-        logger.info(f"The following script is going to be executed:\n" + script.strip())
-        logger.info(f"Script output:")
         stdout = None
         stderr = None
 
+    logger.log(loglevel, f"The following script is going to be executed:\n" + script.strip())
     result = subprocess.run(["/bin/bash", "-c", script_to_run], stdout=stdout, stderr=stderr, cwd=cwd)
+
+    if capture_output:
+        if stdout_loglevel:
+            logger.log(stdout_loglevel, f"STDOUT:")
+            logger.log(stdout_loglevel, try_decode(result.stdout))
+        if stderr_loglevel:
+            logger.log(stdout_loglevel, f"STDERR:")
+            logger.log(stderr_loglevel, try_decode(result.stderr))
```

Comment:
Let's not split `stdout` and `stderr`, it's very frustrating.

Reply:
There's no timing information at this point, do you want me to just output them without specifying which is which?
My rationale was that having stderr clearly separated from stdout might help debugging (as you get stderr at the end, clearly delimited)

Reply:
I understand, but in practice programs interleave `stderr` and `stdout` in a meaningful way and you end up losing that.
Think about a build system printing on `stdout` the file being processed and on `stderr` the error.

Also, if `stderr` is very long and you're interested in stuff printed in `stdout` at the very last you suddenly have to back above `stderr`.

It was like this before and I've faced frustrating situations.